bcm53xx: Specify all RAM for linksys ea6500 v2

alealexpro100 opened the pull request:
This allows to use all memory on both ea6500 v2 and ea6700.

Tested it with file in tmpfs and stress utility. Looks like it works correctly.

It is correction of [this pull request](https://github.com/openwrt/openwrt/pull/10449).

Signed-off-by: Aleksey Nasibulin <alealexpro100@ya.ru>

Ansuel commented:
@alealexpro100 thanks for the pr. you should add a commit description and also use Fixes tag referring to the change related

Ansuel commented:
@alealexpro100 the patch is not good but one last thing space between commit description and sob.

Also would be good something like in the title `extend RAM to full 256MB`

And same in commit description
```
Linksys ea6500-v2 have 256MB of ram. Currently we only use 128MB. Expand the definition
to use all the available RAM.
```

alealexpro100 committed:
bcm53xx: linksys ea6500-v2: extend RAM to full 256MB

Linksys ea6500-v2 have 256MB of ram. Currently we only use 128MB.
Expand the definition to use all the available RAM.

Signed-off-by: Aleksey Nasibulin <alealexpro100@ya.ru>
[ wrap to 80 columns ]
Signed-off-by: Christian Marangi <ansuelsmth@gmail.com>

alealexpro100 commented:
> @alealexpro100 sorry my bad... can you wrap this to 80 columns and this is ready? or if you want i can do it for you.

It would be great if You do so. Thank you!

rmilecki commented:
@alealexpro100: please send this patch for upstream, I really don't want to maintain such trivial patches downstream.

Personally I wouldn't even accept this without having it sent upstream.

Cc @Ansuel

Ansuel commented:
@rmilecki sorry! didn't notice the dts was present upstream!

rmilecki commented:
Ping @alealexpro100, can you send proper patch?

alealexpro100 commented:
> Ping @alealexpro100, can you send proper patch?

What do You mean? Should I send this patch to upstream or edit it?

rmilecki commented:
> What do You mean? Should I send this patch to upstream or edit it?

Please see my previous comment:

> @alealexpro100: please send this patch for upstream, I really don't want to maintain such trivial patches downstream.

alealexpro100 commented:
> > What do You mean? Should I send this patch to upstream or edit it?
> 
> Please see my previous comment:
> 
> > @alealexpro100: please send this patch for upstream, I really don't want to maintain such trivial patches downstream.

I am not familiar with sending patches to upstream process. May be it is possible just remove this patch from a tree?

rmilecki commented:
OK, I'll drop it during the next bump

stintel commented:
> I am not familiar with sending patches to upstream process. May be it is possible just remove this patch from a tree?

https://docs.kernel.org/process/submitting-patches.html

Ansuel commented:
I'm proposing this upstream and they are asking if some config are required like sparse memory support. Can someone help me with that?

Ansuel commented:
Actually the patch was accepted upstream so we need to backport it. It will also eventually backported and later dropped!
